domains/mvhuy.json

```diff
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "huypv2002"
+    },
+    "record": {
+        "RECORDTYPE": "CNAME"
+    }
+}
```

Comment:
File does not follow proper schema. See docs for more information: https://docs.is-a.dev/